### PR TITLE
reg-110 catalog release 2.20.0

### PR DESCRIPTION
**Pull request by ysmendix**
For catalog release 2.20.0, Scheduled on 10th Feb 2022 at 8:30

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@johannahemminger please do a language review for this release tomorrow morning.

File: content/releasenotes/data-hub/index.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ These release notes cover changes made to the [Mendix Data Hub](/data-hub/).
 
 #### New Features
 
-* We added a new **Delete Data Source** option in the **Edit** menu of Data Source.
+* We added a option for users with curation rights to delete a data source. To do this, go to the **Edit** menu of the data source and click **Delete Data Source**.
```
Comment (Adam-Dupaski) [inline content/releasenotes/data-hub/index.md:17]: @johannahemminger please look at article use here for "option".

File: content/releasenotes/data-hub/index.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ These release notes cover changes made to the [Mendix Data Hub](/data-hub/).
...
 
 #### Fixes
 
-* We made improvements to Data Hub security and UI.
-* We fixed a bug where Data Hub Catalog was not displaying the updatable status of an updateable entity
+* We fixed a bug where the Data Hub Catalog did not display the correct status of an updateable entity.
```
Comment (Adam-Dupaski) [inline content/releasenotes/data-hub/index.md:21]: @johannahemminger please search for "updateable" vs. "updatable" for standard usage in current docs.

File: content/releasenotes/data-hub/index.md (hunk 1 above)
Comment (Adam-Dupaski) [inline content/releasenotes/data-hub/index.md:17]: @johannahemminger is there a cross-reference to a doc available here? Or would an additional doc update be necessary for this new feature?

Comment (johannahemminger) [reply content/releasenotes/data-hub/index.md:17]: For now I will add a reference, but will look into updating that doc. There was some discussion in the DH meeting today about the scope of this feature so will look into the implications further.

Comment (Adam-Dupaski) [reply content/releasenotes/data-hub/index.md]: @johannahemminger ok please create a Jira issue for yourself as necessary

File: content/releasenotes/data-hub/index.md [hunk 3]
```diff
@@ -14,11 +14,11 @@ These release notes cover changes made to the [Mendix Data Hub](/data-hub/).
 
 #### New Features
 
-* We added a option for users with curation rights to delete a data source. To do this, go to the **Edit** menu of the data source and click **Delete Data Source**.
+* We added an option to delete a data source, available for users with [curation](data-hub/data-hub-catalog/curate) rights. To delete a data source, go to the [Curation Bar](/data-hub/data-hub-catalog/curate#3-curation-bar) for the data source and click **Edit**, then click **Delete Data Source**.
```
Comment (Adam-Dupaski) [inline content/releasenotes/data-hub/index.md:17]: @johannahemminger this cross-reference throws an error for the Travis build for this PR. Please look closely at syntax used in comparison to Curation Bar cross-reference.

Comment (johannahemminger) [reply content/releasenotes/data-hub/index.md:17]: Slash!

**Review by johannahemminger — APPROVED** [review]

**Comment by johannahemminger** [comment]
Followed Adam's instructions on Slack (with Mark's guidance)

**Pull request merged**